thingsboard_gateway/connectors/ftp/ftp_connector.py

```diff
@@ -415,6 +421,7 @@ def __process_rpc(self, method, value_expression):
                     ftp.storbinary('STOR ' + arr[0], io_stream)
                     io_stream.close()
                     success_sent = True
+                    converted_data = f'The value is written to {arr[0]}' if len(arr) < 80 else True
```

Comment:
Please change this, to print logs, with some string, but RPC should return something like {"result": value}, you can use OPC-UA connector as a reference in this case.